bin/import-borders

```diff
@@ -60,7 +60,10 @@ if [[ "$task" == "import" || "$task" == "parse" ]]; then
   echo "Filtering $IMPORT_PBF_FILE into $BORDERS_PBF_FILE"
   mkdir -p "$(dirname "${BORDERS_PBF_FILE?}")"
   rm -rf "${BORDERS_PBF_FILE?}"
-  osmborder_filter -o "$BORDERS_PBF_FILE" "$IMPORT_PBF_FILE"
+  rm -rf "${BORDERS_PBF_FILE%.pbf}-temp.pbf"
```

Comment:
Please make `"${BORDERS_PBF_FILE%.pbf}-temp.pbf"` into a var and use it here and below. Also, add a new `echo "Dropping broken references (file) -> (file)"`  (or something similar) -- we should indicate the extra step.  The `echo` from above could be moved right before the `osmborder_filter` call.

Also, I do not want this step to always execute, esp when dealing with full planet -- as that would take significant time and won't be much use.  We could introduce a new "task" (command) -- e.g. `import-fix` and `parse-fix` (name suggestions are welcome), or it could be done via an env var.  This way fixing the file would become an optional step.

Lastly, I think this is a temporary workaround, and it should be the `osmborder_filter` that should handle it automatically. Please create an upstream issue in the @pnorman 's https://github.com/pnorman/osmborder

BTW, I think the environment variable path (rather than adding new tasks) would be preferable, because once it is fixed upstream, we can simply ignore that env var.

Reply:
Done